practicals/large_language_models.ipynb

```diff
@@ -2700,44 +2700,25 @@
     },
```

Comment:
<p>This is awesome, thanks Ruan!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='134'/>